deploy/operator/ironic_keepalived/operator_ironic_keepalived.yaml

```diff
@@ -23,28 +22,6 @@ spec:
           envFrom:
             - configMapRef:
                 name: ironic-bmo-configmap
-        - name: baremetal-operator
-          image: quay.io/metal3-io/baremetal-operator
-          ports:
-            - containerPort: 60000
-              name: metrics
-          command:
-            - /baremetal-operator
-          imagePullPolicy: Always
-          env:
-            - name: WATCH_NAMESPACE
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.namespace
-            - name: POD_NAME
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.name
-            - name: OPERATOR_NAME
-              value: "baremetal-operator"
-          envFrom:
-            - configMapRef:
-                name: ironic-bmo-configmap
```

Comment:
let's discuss how to proceed for the backwards compatibility for the operator on the call.